webpack.config.ts

```diff
@@ -1,6 +1,7 @@
 import { ForkTsCheckerWebpackPlugin } from 'fork-ts-checker-webpack-plugin/lib/plugin';
 import path from 'path';
 import { Configuration } from 'webpack';
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
```

Comment:
can we use import statement?

Reply:
@farrah-deriv if you follow @balakrishna-binary 's suggestion and change it to import statement, I suggest adding the type assertion at line 66 🙂  to avoid TS error:
`plugins: [new TsconfigPathsPlugin() as TsconfigPathsPlugin & ResolvePluginInstance],`
I've done it as well in my PR )

Reply:
yup thanks @maryia-deriv 🙂